src/app/util/attribute-storage.cpp

```diff
@@ -261,9 +261,62 @@ uint16_t emberAfGetDynamicIndexFromEndpoint(EndpointId id)
     return kEmberInvalidEndpointIndex;
 }
 
+namespace {
+
+const EmberAfCluster * getClusterTypeDefinition(EndpointId endpointId, ClusterId clusterId, EmberAfClusterMask mask)
```

Comment:
This is called in a single place with mask 0. 

Please apply `YAGNI` here and drop the mask parameter, assume mask is always 0.

Reply:
I am not sure that structure of clusters will be identical between client and server (i.e. attributes may differ).